stock_pack_operation_quick_lot/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+==============================
+Stock Pack Operation Quick Lot
+==============================
+
+This module allows set pack operation Lot Name and End of Life Date directly
+on picking operations
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Inventory > Operations > All transfer
+#. Select a Ready picking or create and Mark as Todo
+#. Set Lot Name [and date] in Operations tab
```

Comment:
Add a new line saying: `#. Set optionally the end of life date in the proper column` and remove `[and date]`, and add a final line:
`#. When the picking will be transferred, the lot will be created with that name and date.`